src/components/screens/BoosterPage/components/BoosterCard.tsx

```diff
@@ -0,0 +1,173 @@
+            completed_boost: "75",
+            active: "65",
+            rating: "3.8"
+        },
+        {
+            name: "Grape",
+            img: "images/default-user-img.jpg",
+            rank: {
+                img: "/images/demo-rank.png",
+                name: "Rank 8",
+            },
+            languages: "EN",
+            completed_boost: "70",
+            active: "60",
+            rating: "3.7"
+        },
+        {
+            name: "Honeydew",
+            img: "images/default-user-img.jpg",
+            rank: {
+                img: "/images/demo-rank.png",
+                name: "Rank 9",
+            },
+            languages: "EN",
+            completed_boost: "65",
+            active: "55",
+            rating: "3.6"
+        },
+        {
+            name: "Iced Melon",
+            img: "images/default-user-img.jpg",
+            rank: {
+                img: "/images/demo-rank.png",
+                name: "Rank 10",
+            },
+            languages: "EN",
+            completed_boost: "60",
+            active: "50",
+            rating: "3.5"
+        },
+        {
+            name: "Jackfruit",
+            img: "images/default-user-img.jpg",
+            rank: {
+                img: "/images/demo-rank.png",
+                name: "Rank 11",
+            },
+            languages: "EN",
+            completed_boost: "55",
+            active: "45",
+            rating: "3.4"
+        }
+    ];
+
+    return (
+        <div className="gap-5 grid grid-cols-3 sm:grid-cols-3">
+            {list.map((item, index) => (
+                <Card shadow="sm" key={index} isPressable onPress={() => console.log("item pressed")}>
+                    <CardBody className="overflow-hidden p-0 h-[350px]">
+                        <Image
+                            shadow="sm"
+                            radius="lg"
+                            width="100%"
+                            alt={item.name}
+                            className="object-cover"
+                            src={item.img}
+                        />
+                    </CardBody>
+                    <CardFooter className="flex-col bg-gradient-to-r from-[#202832] to-[#283347]">
+                        <CardFooter className="justify-between">
+                            <b className="text-white text-3xl">{item.name}</b>
+                            <Image
+                                width="100%"
+                                height="100%"
+                                className="h-[50px] w-[50px]"
+                                src={item.rank.img} alt={item.rank.img}
+                            />
+                            <p className="text-default-500 text-white text-lg font-bold">{item.rank.name}</p>
+                        </CardFooter>
+                        <CardFooter className="justify-between">
+                            <b className="text-white text-lg">Completed Boost</b>
```

Comment:
set text-white lại nhiều quá, e set cho thằng cha to nhất của nó đi r sửa những thằng ko dùng text white thôi